src/Core/Managed/PCL/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -35,5 +35,10 @@ public IDebugOutput GetDebugOutput()
             
             return this.debugOutput;
         }
+
+        public string GetEnvironmentVariable(string name)
+        {
+            return null;
```

Comment:
Good point, I keep thinking of .NET Core as a replacement for PCL but it is completely new and isn't portable backwards (as such).